scylladb/scylla
# doc: Fix broken links

This PR fixes some links that stopped working when changing the domain from https://docs.scylladb.com/ to `https://docs.scylladb.com/<version_name>`

## How to test this PR

1. Run ``make preview``. The build should complete without errors.

2. Open the preview and click on some of the links replaced.
